### DISCUSSION TITLE
Menu icon problem in 1.10.3

### DISCUSSION THREAD

**Discussion by Olifair** [freeplane/freeplane#611] [Help]
I have just installed version 1.10.3 with the Java 17 from the file Freeplane-Setup-with-Java-1.10.3.exe in a windows 10 system. 

And I have problems with many of the drop down menus and icons : most of the them occupy only half a line (see screenshot I added) : the icons to the left mostly seem OK, as do Bold Italics, Strikethrough etc. However the drop down menus for  style, font, font size, and the whole search and filtering part, all menus and icons are only one half line or one quarter of the usual size (for icons). 
Any idea what's going on ? 

![Freeplane1 10 3iconproblem](https://user-images.githubusercontent.com/6613705/187670532-63e437c8-da0f-46fe-b188-6b2ec7a51fdc.PNG)

**Comment by dpolivaev** [comment]
I have found a bug with look and feel "default". It is already fixed in 1.10.4 but this version is still in progress.
Try to choose any other look and feel and write it if helps.

**Comment by Olifair** [comment]
Thanks for your very rapid answer. The "windows" look and fell is a bit better, but still has some of the drop down menus smaller than an acual line height, some more centered to the top, other in the middle or the bottom. I played with the "menu item font size" which seems to change the height of some drop down menus and surprisingly also of some icons/buttons, but not sifficiently.

**Reply by dpolivaev** [reply]
Create an issue with a bug report and attach screenshots with different look-and-feels and your auto.properties

**Reply by Olifair** [reply]
Hmm, this will need some time, might need to wait for the week-end
